SU-HSDO/suhumsci
# STN-526 : Refine Traditional Carousel

# READY FOR REVIEW

## Summary
The work in this PR corrects the alignment of the content and the controls in the traditional carousel.

## Need Review By (Date)
9/17/2020

## Urgency
medium

## Steps to Test
1. In your CLI, run `npm run test` and confirm our tests pass.
2. Set the theme to Traditional 
3. Go to http://sparkbox-sandbox.suhumsci.loc/qa/carousel-hero-image
* If you do not have several examples of the carousel within the main content, you may want to sync with [staging](https://sparkbox-sandbox-stage.stanford.edu/qa/carousel-hero-image).
4. Confirm the following:
- [x] Text does not overflow the carousel but rather the carousel grows to accommodate large blocks of content.
- [x] The carousel controls (dots and arrows) align under the button to allow the button to accommodate a large amount of text without colliding into the carousel controls.
- [x] Spacing between the elements (title, body copy, button) did not change.
5. Switch to the Colorful theme and confirm the update to the traditional themed carousel did not negatively affect the colorful themed carousel.
